.circleci/config.yml

```diff
@@ -775,12 +782,12 @@ jobs:
           version: 19.03.13
       - checkout
       - run:
-         name: Is a release branch
+         name: Is a release or master branch
          command: |
-           # We want to run this job only on release branch, to guess that
+           # We want to run this job only on release or master branch, to guess that
            # we check if a new release entry has been added into the CHANGELOG
            # If there is no new release entry, git diff returns a 0 exit code
-           if git diff --quiet -G"^## \[[0-9.]*\]$" origin/master..HEAD CHANGELOG.md; then
+           if [ "$CIRCLE_BRANCH" != "master" ] | git diff --quiet -G"^## \[[0-9.]*\]$" origin/master..HEAD CHANGELOG.md; then
```

Comment:
Not sure why you need to change this? This will have the effect to run cookie cutter on all master builds which is not what you modified in the above comment?

Reply:
Before your idea to add a nightly round, I first updated circle ci configuration to run this job on master branch. I kept this change then I added a new commit to setup the nightly round.

Reply:
yes but my point is that the comment above should be:
```
# We want to run this job only when a release is merged to the master branch, to guess that
```

Reply:
Oh ok!!

Reply:
I updated the condition. No I stop the job if the branch is not master and there is no release commit.